src/angularjs/src/styles/layout/_footer.scss

```diff
@@ -1,12 +1,12 @@
 footer {
   @extend %clearfix;
   display: block;
-  background-color: #000;	
+  background-color: #002c40;	
```

Comment:
Since this color is used in several places, how about defining it in `_variables.scss`?